<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu_hackProgresser</class>
 <widget class="QDialog" name="menu_hackProgresser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>412</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>372</width>
    <height>158</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>372</width>
    <height>412</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NASA Ultron Connecting Sentinel v.6.9</string>
  </property>
  <widget class="QLabel" name="targetLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="currentProgress">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>361</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="overallProgress">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>361</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="currentLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-2</number>
   </property>
   <property name="text">
    <string>Current Progress</string>
   </property>
  </widget>
  <widget class="QLabel" name="overallLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Overall Progess</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hacking . . .</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="detailsBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>351</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="detailsButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Details</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
